com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LayeringOptionsUIBlock.cs

```diff
@@ -76,21 +75,14 @@ public override void LoadMaterialProperties()
         }
 
         /// <summary>
-        /// Renders the properties in the block.
+        /// Property that specifies if the scope is a subheader
         /// </summary>
-        public override void OnGUI()
-        {
-            // We're using a subheader here because we know that layering options are only used within layers
-            using (var header = new MaterialHeaderScope(Styles.header, (uint)m_ExpandableBit, materialEditor, subHeader: true))
-            {
-                if (header.expanded)
-                {
-                    DrawLayeringOptionsGUI();
-                }
-            }
-        }
+        protected override bool isSubHeader => true;
```

Comment:
why is this class specifically a subheader?

Reply:
see my other comment please